services/brig/src/Brig/API/Public.hs

```diff
@@ -221,14 +224,24 @@ type OutsideWorldAPI =
     :<|> GetUserUnqualified
     :<|> GetUserQualified
 
-type ServantAPI =
-  "brig" :> "api-docs" :> Get '[Servant.JSON] Swagger
-    :<|> OutsideWorldAPI
+type ServantHandlerAPI = "api" :> SwaggerSchemaUI "swagger-ui" "swagger.json"
+
+type ServantAPI = OutsideWorldAPI
```

Comment:
The names `ServantAPI` and `ServantHandlerAPI` don't communicate much difference to me. Can we rename `ServantHandlerAPI` to something more meaningful, maybe `SwaggerDocsAPI`?

Reply:
good point! Thanks.